starter-code/css/main.css

```diff
@@ -7,4 +7,98 @@ light blue: #1c5380
 body {
   font: 200 16px/1.5 "Helvetica Neue", Helvetica, Arial, sans-serif;
   margin: 0;
+  background-image : linear-gradient(to bottom, #1c5380 223px, #fff 203px);
 }
+
+.container{
+ 
+  display: flex;
+  justify-content: space-around;
+  width: 70%;
+  margin: 30px auto;
+}
+
+.img-left{
+  
+  padding: 20px;
```

Comment:
Padding is usually used when you want to separate content from the edges of its container, but more in a visual way. If what you want is to move your content, reducing / increasing the size of the container is a better option.